lib/workers/branch/auto-replace.spec.ts

```diff
@@ -1,24 +1,36 @@
-import { defaultConfig, loadFixture } from '../../../test/util';
+import { Fixtures } from '../../../test/fixtures';
+import { defaultConfig } from '../../../test/util';
+import { GlobalConfig } from '../../config/global';
 import { WORKER_FILE_UPDATE_FAILED } from '../../constants/error-messages';
 import { extractPackageFile } from '../../manager/html';
 import type { BranchUpgradeConfig } from '../types';
 import { doAutoReplace } from './auto-replace';
 
-const sampleHtml = loadFixture('sample.html', `../../manager/html`);
+const sampleHtml = Fixtures.get('sample.html', `../../manager/html`);
 
-jest.mock('../../util/fs');
+jest.mock('fs-extra', () => Fixtures.fsExtra());
 
 describe('workers/branch/auto-replace', () => {
   describe('doAutoReplace', () => {
     let reuseExistingBranch: boolean;
     let upgrade: BranchUpgradeConfig;
+    beforeAll(() => {
+      GlobalConfig.set({
+        localDir: '/temp',
+      });
+    });
     beforeEach(() => {
       upgrade = {
         ...JSON.parse(JSON.stringify(defaultConfig)),
         manager: 'html',
+        packageFile: 'test',
       };
       reuseExistingBranch = false;
     });
+    afterAll(() => {
+      GlobalConfig.reset();
+    });
+
```

Comment:
I don't think that this is required.

Reply:
Why not? It is a global object, it can affect another test

Reply:
Nope, it can only affect tests in this file. Jest will isolated each file with fresh modules.
You only need to revert build-in object changes, like our `nock` tools does.

So please remove this.